# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libdvdread/inttypes_h.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libdvdread-0.9.5/dvdread/dvd_reader.h.orig	2006-03-18 13:36:18.000000000 +0100
+++ libdvdread-0.9.5/dvdread/dvd_reader.h	2006-03-18 13:33:36.000000000 +0100
@@ -23,6 +23,7 @@
  */
 
 #include <sys/types.h>
+#include <inttypes.h>
 
 /**
  * The DVD access interface.
